### PR TITLE
Added transformation from Phred 64 to Phred 33

### PR DESCRIPTION
**Pull request by JenkoB**

### CODE REVIEW

**Comment by JenkoB** [comment]
@jkokosar please review

File: resolwe_bio/tests/processes/test_upload.py [hunk 1]
```diff
@@ -80,6 +80,7 @@ def test_upload_cuffquant_expr(self):
         self.run_process('upload-expression-cuffnorm', inputs)
 
     def test_upload_paired_end_reads(self):
+        self.keep_data()
```
Comment (mstajdohar) [inline resolwe_bio/tests/processes/test_upload.py]: This should be removed.

File: resolwe_bio/processes/import_data/seq_reads.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@
   requirements:
     expression-engine: jinja
   data_name: '{{ src.0.file|default("?") }}'
-  version: 1.0.22
+  version: 1.0.23
```
Comment (mstajdohar) [inline resolwe_bio/processes/import_data/seq_reads.yml]: Increase the middle version number.

File: resolwe_bio/processes/import_data/seq_reads.yml [hunk 3]
```diff
@@ -85,7 +91,7 @@
   requirements:
     expression-engine: jinja
   data_name: '{{ src1.0.file|default("?") }}'
-  version: 1.0.22
+  version: 1.0.23
```
Comment (mstajdohar) [inline resolwe_bio/processes/import_data/seq_reads.yml]: Increase the middle version number.

File: resolwe_bio/processes/import_data/seq_reads.yml [hunk 4]
```diff
@@ -160,14 +161,53 @@
...
-        fi
+      NAMES_FW=($FW_FASTQC)
+
+      for name in ${NAMES_FW[@]}; do
+        READS_NAME_FW=`basename $name _fastqc.zip`
+
+        for report_dir in ./fastqc/"${READS_NAME_FW}_fastqc"/; do
+          ENCODING=`parse_encoding_type.py ${report_dir}fastqc_data.txt`
+
+          if [ "${ENCODING}" == "Illumina 1.5" ] ||  [ "${ENCODING}" == "Illumina 1.3" ]; then
```
Comment (mstajdohar) [inline resolwe_bio/processes/import_data/seq_reads.yml]: This IF statement is the same as for the single ended reads. We should write a bash script that checks the encoding version and does the transformation. The we should call this script from both processes.

**Pull request merged**